specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/CommonDefinitions.json

```diff
@@ -1149,6 +1159,20 @@
           ]
         }
       }
+    },
+    "ServiceBind": {
+      "definition": "List of services bound to the app",
+      "type": "object",
+      "properties": {
+        "serviceId": {
+          "description": "Id of the service",
```

Comment:
is it a resource id?

Reply:
Yes, it's the resourceId. Updated the description to match.